### PR TITLE
Adding PV Surrogate

### PR DESCRIPTION
**Pull request by zacharybinger**
* Adding PVSurrogate class that handles creation and loading of the PV surrogate
* Surrogate has 1 input (Design Size) and 2 outputs (Annual Energy, Land Required)

### CODE REVIEW

**Review by kurbansitterley — APPROVED** [review]
LGTM

File: src/watertap_contrib/reflo/solar_models/surrogate/pv/pv_surrogate.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        self.annual_energy = Var(
+            initialize=1,
+            units=pyunits.kWh,
+            doc="annual energy produced by the plant in kWh",
+        )
+
+        self.land_req = Var(
+            initialize=7e7,
+            units=pyunits.acre,
+            doc="annual energy produced by the plant in kWh",
```
Comment (mjprilliman) [inline src/watertap_contrib/reflo/solar_models/surrogate/pv/pv_surrogate.py:69]: "Land area required by the plant in acres"

File: src/watertap_contrib/reflo/solar_models/surrogate/pv/pv_surrogate.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+            initialize=1000,
+            bounds=[1, 200000],
+            units=pyunits.kW,
+            doc="PV design size in kW",
+        )
+
+        self.annual_energy = Var(
+            initialize=1,
+            units=pyunits.kWh,
+            doc="annual energy produced by the plant in kWh",
```
Comment (mjprilliman) [inline src/watertap_contrib/reflo/solar_models/surrogate/pv/pv_surrogate.py:63]: Should be consistent with trough_surrogate.py and capitalize "Annual"

Comment (zacharybinger) [reply src/watertap_contrib/reflo/solar_models/surrogate/pv/pv_surrogate.py]: fixed

File: src/watertap_contrib/reflo/solar_models/surrogate/pv/pv_surrogate.py (hunk 1 above)
Comment (zacharybinger) [reply src/watertap_contrib/reflo/solar_models/surrogate/pv/pv_surrogate.py]: fixed

**Review by mjprilliman — APPROVED** [review]

**Pull request merged**